Client/Frontend/LoginManagement/LoginListViewController.swift

```diff
@@ -428,6 +428,8 @@ extension LoginListViewController: UITableViewDelegate {
     }
 
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        // Check that section is > 0, which means that a Login Record is selected, otherwise we do not want to select the row since the returned Login Record is nil and the assertion failure function will cause a crash
+        guard indexPath.section > 0 else { return }
```

Comment:
This is okay, but is it possible to check why this cell appears in the first place and prevent that?

Reply:
Hey, I did check why this cell appears here and we want to preserve that fact that this cell is displayed.  It is a different type of cell: `LoginListTableViewSettingsCell` . It is only used in Logins Settings section that contains only settings with toggles (as in the attached screenshot), so the rows should not be selectable. I changed the solution in order to prevent it from the beginning to be selectable as you suggested. The cell should appear here, but should not be selectable. :)  I hope this solution is ok. 

<img width="565" alt="Screenshot 2023-05-02 at 5 30 39 PM" src="https://user-images.githubusercontent.com/46751540/235698704-320dbbd9-539b-4ca3-83b9-4ec546f44df5.png">